cli/cmd/install.go

```diff
@@ -397,7 +397,7 @@ func render(w io.Writer, values *l5dcharts.Values, stage string, options valuesp
 	}
 
 	if stage == "" || stage == controlPlaneStage {
-		overrides, err := renderOverrides(values, values.GetGlobal().Namespace)
+		overrides, err := renderOverrides(values, values.GetGlobal().Namespace, false)
```

Comment:
Is there a reason that we don't want to just stringData always? is it the case that `values` is already b64 encoded here?

Reply:
As far as I can tell, not really a compelling reason one way or the other.  APIMachinery handles the b64 encoding for us, so it's really just a matter of choosing between `string` and `[]byte`.  

I may update this in a follow up to always use stringData, since I don't really want to clutter up this PR with the resulting golden file changes.